resources/govuk-template.mustache.html

```diff
@@ -424,11 +424,10 @@
 
                                 {{/isVerify}}
 
-
-                                <div class="{{^isGovernmentGateway}}{{^isVerify}} column-half {{/isVerify}}{{/isGovernmentGateway}}
-                                {{#isGovernmentGateway}}{{^isSa}} column-half {{/isSa}}{{/isGovernmentGateway}}
+                                {{^isVerify}}
+                                <div class="{{#isGovernmentGateway}}{{^isSa}} column-half {{/isSa}}{{/isGovernmentGateway}}
```

Comment:
Also need to add back in ^GG
```suggestion
                                <div class="{{^isGovernmentGateway}}column-half {{/isGovernmentGateway}}
```

Reply:
Keep current code there too! This is additional line